lib/main.dart

```diff
@@ -4,6 +4,7 @@ import 'dart:convert';
 import 'package:comiko/app_state.dart';
 import 'package:comiko/pages/upcoming_events_page.dart';
 import 'package:comiko/services.dart';
+import 'package:comiko/widgets/LocationGPS.dart';
```

Comment:
All our package import should be after third-party imports.

Reply:
https://www.jetbrains.com/help/idea/optimizing-imports.html